spec/index.html

```diff
@@ -5669,248 +5731,221 @@ <h5>NOT IN</h5>
+                `"NaN"^^xsd:double` and `"NaN"^^xsd:float` are considered to 
+                represent the same value.
+                If <code>term1</code> and <code>term2</code> are
+                both "NaN" for either xsd:double or xsd:float, then 
+                return TRUE.
+              </li>
+              <li>If <code>term1</code> and <code>term2</code> are both 
+                <a data-cite="RDF12-CONCEPTS#dfn-literal">literals</a>
+                and the SPARQL processor can determine that their the values are equal,
+                then return TRUE.
+              </li>
+              <li>If <code>term1</code> and <code>term2</code> are both 
+                <a data-cite="RDF12-CONCEPTS#dfn-literal">literals</a>
+                and the SPARQL processor can determine the values can not be equal,
+                then return FALSE.
               </li>
               <li>
-                <span class="name">term1</span> and <span class="name">term2</span> are the same
-                <span class="bnode type">blank node</span>.
+                If <code>term1</code> and <code>term2</code> are both 
+                <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a>,
+                apply the function `sameValue` pair-wise to each of the components.
+                Return TRUE if each component pair returns TRUE;
+                produce a type error if any component pair produces an error;
+                otherwise return FALSE.
               </li>
-            </ul>
-            <p id="func-RDFterm-equal-note1" class="note">
+              <li>Otherwise, return FALSE.</li>
+            </ol>
+            <div id="sameValue-ill-typed" class="note">
+              <p>A literal is 
+                <a data-cite="RDF12-CONCEPTS#dfn-ill-typed">ill-typed</a>
+                if its datatype is handled by the SPARQL processor and
+                its lexical form is not in the
+                <a data-cite="RDF12-CONCEPTS#dfn-lexical-space">lexical space</a>
+                of the datatype.
+              </p>
+              <p>
+                If the two arguments are literals, the function `sameValue` 
+                returns `true` or `false` in cases where the SPARQL processor
+                can determine that the values of these literals are equal or are not equal. 
+                If the SPARQL processor can not be sure, it returns `error`.
+              </p>
+            </div>
+            <div id="sameValue-NaN" class="note">
+              <p>
+                The <a href="#OperatorMapping">Operator Mapping</a> for "`=`"
+                is the function
+                <a data-cite="XPATH-FUNCTIONS-31#func-numeric-equal">`fn:numeric-equal`</a>
+                which returns `false` when comparing arguments involving `NaN`.
+                However, `sameTerm("NaN"^^xsd:double, "NaN"^^xsd:double)` is true;
+                `sameValue` treats "NaN"^^xsd:double as the "same value".
```

Comment:
Some minor suggestions to the phrasing:
```suggestion
                The <a href="#OperatorMapping">Operator Mapping</a> for "`=`"
                uses the function
                <a data-cite="XPATH-FUNCTIONS-31#func-numeric-equal">`fn:numeric-equal`</a>
                which is defined to return `false` when comparing arguments involving `NaN`.
                However, since equality of terms is given precedence over `fn:numeric-equal`,
                and `sameTerm("NaN"^^xsd:double, "NaN"^^xsd:double)` is true;
                `sameValue` treats "NaN"^^xsd:double as the "same value".
```

Reply:
The new line 5898 is not a complete sentence, which it should be. 

There should be a trailing `as xyz` that says what it is that ``` `"NaN"^^xsd:double``` is `treated as the "same value"` *of*.

To be maybe a bit clearer, ``` `sameValue` treats "NaN"^^xsd:double as the "same value" as _xyz_.```

Reply:
Incorporated the first part of @rubensworks suggestion : "is defined to return"

The latter part revised to 

```
                However, `sameTerm("NaN"^^xsd:double, "NaN"^^xsd:double)` is true.
                The function `sameValue` defines `sameValue("NaN"^^xsd:double, "NaN"^^xsd:double)`
                to be true because the arguments are the same element of the value space.
```